File: server/handle.py
import requests  # 导入 requests 库
import time  # 导入 time 库

url = "http://www.106u.com/ajax.php"

headers = {
    "Accept": "application/json, text/javascript, */*; q=0.01",
    "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8",
    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36",
    "X-Requested-With": "XMLHttpRequest",
    "Origin": "http://www.106u.com",
    "Referer": "http://www.106u.com/reg.php",
}


# 验证账号是否存在
def check_account(username):
    data = {"LOGIN_ACCOUNT": username, "action": "checkUser", "username": username}
    response = requests.post(url, headers=headers, data=data)
    return response.text
    pass


# 注册账号
def register_account(username, password, idcard, truename):
    # 注册请求示例（对应最后一个entry）
    data = {
        "action": "register",
        "post_mode": "withtml5",
        "sp": "",
        "username": username,
        "password": password,
        "passwordconfirm": password,
        "email": username + "@qq.com",
        "truename": truename,  # 原值为URL编码的%E5%AD%99%E6%A0%91%E5%8D%9A
        "idcard": idcard,
    }

    response = requests.post(url, headers=headers, data=data)
    return response.text

    # http://www.106u.com/game.php?action=play&game_id=438&server_id=122566


if __name__ == "__main__":
    truename = "孙树博"  # 替换为实际的用户名
    idcard = "132201198505063774"  # 替换为实际的用户名
    password = "1q2w3e4r."  # 替换为实际的密码
    account_ext = "test"
    for i in range(1000, 1100):
        acc_resp = check_account(account_ext + str(i))
        reg_resp = register_account(account_ext + str(i), password, idcard, truename)
        # 添加 2 秒延迟
        time.sleep(3)
        print(account_ext + str(i) + ": ", acc_resp, reg_resp)
        pass

File: html_moli.yofijoy.com/handle/handle.js


// const url = "http://www.106u.com/ajax.php";
const url = "http://127.0.0.1/106u/ajax.php";

let stopFlag = false; // 创建控制器变量

const headers = {
    "Accept": "application/json, text/javascript, */*; q=0.01",
    "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8",
    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36",
    "X-Requested-With": "XMLHttpRequest",
    "Origin": "http://www.106u.com",
    "Referer": "http://www.106u.com/reg.php",
};

// 验证账号是否存在
async function check_account(username) {
    const data = {
        LOGIN_ACCOUNT: username,
        action: "checkUser",
        username: username
    };
    const urlEncodedData = new URLSearchParams(data).toString();
    try {
        const response = await fetch(url, {
            method: 'POST',
            headers: headers,
            body: urlEncodedData
        });
        return await response.json();
    } catch (error) {
        console.error('Error checking account:', error);
        return null;
    }
}

// 注册账号
async function register_account(username, password, idcard, truename) {
    const data = {
        action: "register",
        post_mode: "withtml5",
        sp: "",
        username: username,
        password: password,
        // 以下是当前选中代码部分，使用 fetch 替代 axios
        passwordconfirm: password,
        email: username + "@qq.com",
        truename: truename,
        idcard: idcard
    };
    const urlEncodedData = new URLSearchParams(data).toString();
    try {
        const response = await fetch(url, {
            method: 'POST',
            headers: headers,
            body: urlEncodedData
        });
        return await response.json();
    } catch (error) {
        console.error('Error registering account:', error);
        return null;
    }
}

// 准备账号信息
async function prepare_account(account, password, idcard, truename, start, end) {
    for (let i = start; i <= end; i++) {
        if (stopFlag) {
            stopFlag = false; // 重置标志，以便下次使用
            break;
        }
        accountStatus.innerText = "正在准备账号：" + account + i;
        await new Promise(resolve => setTimeout(resolve, 3000));
        let resp = await check_account(account + i);
        console.log(resp);
    }
}

// 停止定时循环的方法
function stopPrepareAccount() {
    stopFlag = true;
}